Library/Homebrew/cmd/fetch.rb

```diff
@@ -64,7 +65,7 @@ def fetch
   def fetch_bottle?(f)
     return true if ARGV.force_bottle? && f.bottle
     return false unless f.bottle && f.pour_bottle?
-    return false if ARGV.build_from_source? || ARGV.build_bottle?
+    return false if FormulaInstaller.build_formula_from_source?(f)
```

Comment:
Okay, gotcha.

I think `ARGV` is pretty good, since it's mostly testing the combination of options and arguments in `ARGV`. The fact that it has several `ARGV` references inside it is an indicator, and that disappears once it's moved to `ARGV`.

Moved it there.